validator/validators/validator.go

```diff
@@ -9,11 +9,14 @@ import (
 	"time"
 
 	"github.com/aws/amazon-cloudwatch-agent-test/validator/models"
+	"github.com/aws/amazon-cloudwatch-agent-test/validator/validators/performance"
 	"github.com/aws/amazon-cloudwatch-agent-test/validator/validators/stress"
 )
 
 func NewValidator(vConfig models.ValidateConfig) (validator models.ValidatorFactory, err error) {
 	switch vConfig.GetValidateType() {
+	case "performance":
```

Comment:
The validate type are being passed as [part of the yaml](https://github.com/aws/amazon-cloudwatch-agent-test/pull/151/files#diff-4182821602cec8db84c8b29427a65c5a63beff4c19bcf93e305966bed05a721eR4). In this case, its performance.